src/pages/Main/MainSlide/MainSlide.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import MainSlideItem from './MainSlideItem/MainSlideItem';
+
+function MainSlide() {
+  return (
+    <SlideBox>
+      <ArrowBtn className="left">
```

Comment:
```suggestion
      <ArrowBtn direction="left">
```
이렇게 혹은
```suggestion
      <ArrowBtn left>
```
이런 형태로 작성한 뒤 props로 받아 쓸 수 있도록 해주세요